<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>438</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>901</width>
    <height>438</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>901</width>
    <height>438</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Senai Excel Management [Delete]</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label_path">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>CAMINHO DO ARQUIVO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_plan">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>350</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
color: black;
border:none;
border-radius: 4px
}

QPushButton:hover{
background-color: rgb(148,148, 148);
border: none;
}</string>
   </property>
   <property name="text">
    <string>ATUALIZAR PLANILHA</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo_senai">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>551</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>900</width>
     <height>200</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/logo-senai.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="load_excel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>200</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
color: black;
border:none;
border-radius: 4px
}

QPushButton:hover{
background-color: rgb(148,148, 148);
border: none;
}</string>
   </property>
   <property name="text">
    <string>CARREGAR ARQUIVO</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_msg">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 3px;</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="load_path">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 4px;
border: none;</string>
   </property>
   <property name="text">
    <string>Example: 'C:/User/Documents/file.xlsx...'</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_path_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SELECIONE A LINHA</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_line">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: none;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_deletar">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
color: black;
border:none;
border-radius: 4px
}

QPushButton:hover{
background-color: rgb(148,148, 148);
border: none;
}</string>
   </property>
   <property name="text">
    <string>DELETAR</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>861</width>
     <height>291</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>951</width>
     <height>461</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <zorder>tableWidget</zorder>
  <zorder>label_path</zorder>
  <zorder>update_plan</zorder>
  <zorder>logo_senai</zorder>
  <zorder>load_excel</zorder>
  <zorder>lbl_msg</zorder>
  <zorder>load_path</zorder>
  <zorder>label_path_4</zorder>
  <zorder>cb_line</zorder>
  <zorder>botao_deletar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
